ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml

```diff
@@ -94,23 +94,12 @@ test_binary_build_commands: make build/operator-sdk-dev-x86_64-linux-gnu
 tests:
 - as: unit
   commands: |
-    mkdir -p $GOPATH/src/github.com/operator-framework/operator-sdk
-    cp -r ./* $GOPATH/src/github.com/operator-framework/operator-sdk/
-    cd $GOPATH/src/github.com/operator-framework/operator-sdk
-    rm -rf vendor/github.com/operator-framework/operator-sdk
     make test/unit
```

Comment:
Do we still have `test/unit`. I thought we changed to `test-unit`?

Reply:
Fixed with latest update. Excellent catch.